### PR TITLE
Added Customer Contact API

### PR DESCRIPTION
**Pull request by omenocal**
For more information visit the official docs:
https://developer.amazon.com/docs/custom-skills/request-customer-contact-information-for-use-in-your-skill.html

### CODE REVIEW

File: docs/customerContact.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  });
+
+Voxa also has a method to request all parameters at once:
+
+.. code-block:: javascript
+
+  const skill = new Voxa({ variables, views });
+
+  skill.onIntent('FullAddressIntent', async (alexaEvent) => {
+    const info = await alexaEvent.customerContact.getFullUserInformation();
```
Comment (armonge) [inline docs/customerContact.rst]: seems like `getFullUserInformation` is missing from the API described in this documentation

Comment (omenocal) [reply docs/customerContact.rst]: True that

File: lib/apis/CustomerContact.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  getPhoneNumber() {
+    return rp(this.getOptions('v2/accounts/~current/settings/Profile.mobileNumber'));
+  }
+
+  /*
+   * Gets user's full contact information
+   * https://developer.amazon.com/docs/custom-skills/request-customer-contact-information-for-use-in-your-skill.html#get-customer-contact-information
+   */
+  async getFullUserInformation() {
+    const info = {
```
[the remark this answers is not in the document]
Comment (rmberrios) [reply lib/apis/CustomerContact.js]: Try using Promise.all[]

File: lib/apis/CustomerContact.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+   */
+  async getFullUserInformation() {
+    const infoRequests = [
+      this.getEmail(),
+      this.getGivenName(),
+      this.getName(),
+      this.getPhoneNumber(),
+    ];
+
+    const result = await Promise.all(infoRequests);
```
Comment (armonge) [inline lib/apis/CustomerContact.js]: `const [ email, givenName, name] = await Promise.all(infoRequests);`

File: lib/apis/CustomerContact.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    return rp(this.getOptions('v2/accounts/~current/settings/Profile.givenName'))
+      .catch(() => undefined);
+  }
+
+  /*
+   * Gets user's name
+   * https://developer.amazon.com/docs/custom-skills/request-customer-contact-information-for-use-in-your-skill.html#get-customer-contact-information
+   */
+  getName() {
+    return rp(this.getOptions('v2/accounts/~current/settings/Profile.name'))
```
Comment (armonge) [inline lib/apis/CustomerContact.js]: I'm not sure if we should be ignoring all possible errors. Accorging to https://developer.amazon.com/docs/custom-skills/request-customer-contact-information-for-use-in-your-skill.html#possible-responses the only error code we can safely ignore is `403 Unauthorized` which means the user didn't give permission

**Pull request merged**